### PR TITLE
Changes for date input value is overlapping with formatted value

### PR DESCRIPTION
**Pull request by lakshmisuhas1996**

### CODE REVIEW

File: src/global.scss [hunk 1]
```diff
@@ -458,12 +458,14 @@ ref - https://stackoverflow.com/a/31162426
...
   &__format{
     position: relative;
+    color: transparent;
 
     &::before {
       position: absolute;
       left: 4px;
       content: attr(data-date);
       display: inline-block;
+      color: black;
```
Comment (iamtarun1993) [inline src/global.scss:468]: color: #000;

File: src/global.scss [hunk 2]
```diff
@@ -458,12 +458,14 @@ ref - https://stackoverflow.com/a/31162426
...
   &__format{
     position: relative;
+    color: transparent;
 
     &::before {
       position: absolute;
       left: 4px;
       content: attr(data-date);
       display: inline-block;
+      color: #000;
```
Comment (abhishek1234321) [inline src/global.scss:468]: please use a SCSS variable

File: src/global.scss [hunk 3]
```diff
@@ -458,12 +458,14 @@ ref - https://stackoverflow.com/a/31162426
 .date-input {
   &__format{
     position: relative;
+    color: transparent;
```
Comment (abhishek1234321) [inline src/global.scss:461]: can we use a hex code here?

Comment (lakshmisuhas1996) [reply src/global.scss:461]: Have tried with rgb & rgba codes here.. Only setting color to transparent and white working here?

**Review by abhishek1234321 — APPROVED** [review]

**Pull request merged**